do-catalog/src/views/DatasetData.vue

```diff
@@ -186,27 +186,18 @@ export default {
     showTooltip(variableName, event) {
       let tooltipInfo = this.findVariableInfo(variableName);
       let tableBoundingSize = this.$refs.tableWrapper.getBoundingClientRect();
-      // let targetBoundingSize = event.target.getBoundingClientRect();
-      // this.tooltip.left = targetBoundingSize.left - tableBoundingSize.left;
       this.tooltip.left = event.clientX - 22;
       if (this.tooltip.left < 140) {
         this.tooltip.isFirst = true;
-        this.tooltip.left -= 26
+        this.tooltip.left -= 26;
```

Comment:
Tooltips are broken in IE11 and Linux Chrome and Firefox at least:
![image](https://user-images.githubusercontent.com/938057/84890895-56266e00-b09b-11ea-8cdb-54a5a153a6eb.png)

Reply:
Can we open another issue in GitHub to solve it? @Jesus89

Reply:
Yes. @xavipereztr has already created a ticket with that issue

Reply:
Previous tooltips seemed to work well (at least in OSX Chrome, IE11 and Linux Chrome and Firefox), what did you found to change it?